vxquery-xtest/src/test/resources/cat/TestRewriteRules.xml

```diff
@@ -0,0 +1,28 @@
+<!--
```

Comment:
I thought you removed this test group. It looks like you have two or three sets of new tests.